### PR TITLE
added metaAndAssetCtxs in info.py and the corresponding type in types.py

### PR DESCRIPTION
**Pull request by ito69**
added method for metaAndAssetCtxs endpoint in info

### CODE REVIEW

File: hyperliquid/info.py [hunk 1]
```diff
@@ -168,6 +168,47 @@ def meta(self) -> Meta:
...
+                    "premium": str,
+                    "prevDayPx": str
+                },
+                ...
+            ]
+
+
+        """
+        return cast(MetaAndAssetCtxs, self.post("/info", {"type": "metaAndAssetCtxs"}))  
+        # return self.post("/info", {"type": "metaAndAssetCtxs"})
```
Comment (lmlmt) [inline hyperliquid/info.py:210]: nit: remove commented out code

**Review by lmlmt — APPROVED** [review]

File: hyperliquid/info.py (hunk 1 above)
Comment (Narenthirasamy) [reply hyperliquid/info.py]: Does this code have ask price and bid price??

**Comment by zknpr** [comment]
Please add "MetaAndAssetCtxs" to the sdk

**Comment by lmlmt** [comment]
Closing PR in favor of recently merged PR which includes this change https://github.com/hyperliquid-dex/hyperliquid-python-sdk/pull/45

**Pull request closed without merging**